pkg/helmclient/install.go

```diff
@@ -16,7 +16,7 @@ func (c *Client) InstallReleaseFromTarball(ctx context.Context, chartPath, names
 
 	t := prometheus.NewTimer(histogram.WithLabelValues(eventName))
 	defer func() {
-		eventCounter.WithLabelValues(eventName).Inc()
+		eventCounter.WithLabelValues(eventName, options.ReleaseName).Inc()
```

Comment:
Getting the release name from `options` struct. Not sure why it is in the `options` struct, unlike other commands.

Reply:
That's how its done in the helm action package. Not sure why. Possibly because helm 2 auto generated names but now you have to provide one.